<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientCrawler</class>
 <widget class="QWidget" name="ClientCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientCrawler</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Adresse">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Port">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quitter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse du serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../images/arrow_up.png</normaloff>../../images/arrow_up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Right">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../images/arrow_right.png</normaloff>../../images/arrow_right.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../images/arrow_left.png</normaloff>../../images/arrow_left.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Down">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../images/arrow_down.png</normaloff>../../images/arrow_down.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetInformations">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientCrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
